### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wintere**
Clarified BugSwarm spawner README.md instructions after testing setup procedures.

### CODE REVIEW

**Comment by dtomassi** [comment]
@lxylxy123456 Please take a look at this.

**Comment by lxylxy123456** [comment]
I think I will rework this when merging this `spawner/README.md` into `README.md`.

**Comment by lxylxy123456** [comment]
@wintere Have you tried "Run Docker Inside Docker"? Does it work well?

File: spawner/README.md [hunk 1]
```diff
@@ -39,20 +27,24 @@ There are two ways to run the spawner.
...
+
+### Setting up environment
+1. Inside the container, pull the git repository
 	```sh
-	$ ./provision.sh
+	$ git pull
 	```
-1. Continue with any other commands (e.g. `./run_reproduce_project.sh`)
-
+1. Proceed with the database setup and provisioning instructions described [here](https://github.com/BugSwarm/bugswarm).
```
Comment (lxylxy123456) [inline spawner/README.md:50]:
```suggestion
1. Proceed with the database setup and provisioning instructions described [here](../README.md).
```

Comment (wintere) [reply spawner/README.md:50]: Now that we've already made merged the Spawner instructions into the main ones onto the dev repo, I'll move the clarity related changes I made to that branch.

**Pull request closed without merging**